### YamlMime:ManagedReference
items:
- uid: System.Activities.UnhandledExceptionAction
  id: UnhandledExceptionAction
  children:
  - System.Activities.UnhandledExceptionAction.Abort
  - System.Activities.UnhandledExceptionAction.Cancel
  - System.Activities.UnhandledExceptionAction.Terminate
  langs:
  - csharp
  name: UnhandledExceptionAction
  nameWithType: UnhandledExceptionAction
  fullName: System.Activities.UnhandledExceptionAction
  type: Enum
  summary: "Specifica l&quot;azione che si verifica quando un&quot;eccezione ignora la radice di un flusso di lavoro."
  remarks: "Il <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>funzione viene richiamata se un&quot;eccezione non raggiunge la radice del flusso di lavoro.</xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> Il <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs>fornisce l&quot;accesso per l&quot;eccezione nonché un puntatore al <xref:System.Activities.Activity>che ha generato l&quot;eccezione.</xref:System.Activities.Activity> </xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> Terminare è l&quot;azione predefinita se non <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>gestore viene specificato.</xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>"
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is terminated.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#6](~/add/codesnippet/csharp/t-system.activities.unha_1.cs)]"
  syntax:
    content: public enum UnhandledExceptionAction
  inheritance:
  - System.Enum
  platform:
  - net462
- uid: System.Activities.UnhandledExceptionAction.Abort
  id: Abort
  parent: System.Activities.UnhandledExceptionAction
  langs:
  - csharp
  name: Abort
  nameWithType: UnhandledExceptionAction.Abort
  fullName: System.Activities.UnhandledExceptionAction.Abort
  type: Field
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Specifica che il <xref href=&quot;System.Activities.WorkflowApplication&quot;> </xref> deve interrompere il flusso di lavoro."
  remarks: "Di conseguenza, <xref:System.Activities.WorkflowApplication.Aborted%2A>chiamato una volta completato il processo di interruzione.</xref:System.Activities.WorkflowApplication.Aborted%2A> L&quot;eccezione non gestita viene utilizzata come motivo di interruzione."
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is aborted.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#7](~/add/codesnippet/csharp/t-system.activities.unha_2.cs)]"
  syntax:
    content: Abort
    return:
      type: System.Activities.UnhandledExceptionAction
      description: "Da aggiungere."
  exceptions: []
  platform:
  - net462
- uid: System.Activities.UnhandledExceptionAction.Cancel
  id: Cancel
  parent: System.Activities.UnhandledExceptionAction
  langs:
  - csharp
  name: Cancel
  nameWithType: UnhandledExceptionAction.Cancel
  fullName: System.Activities.UnhandledExceptionAction.Cancel
  type: Field
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Specifica che il <xref href=&quot;System.Activities.WorkflowApplication&quot;> </xref> deve pianificare l&quot;annullamento dell&quot;attività radice e riprendere l&quot;esecuzione."
  remarks: "Di conseguenza, <xref:System.Activities.WorkflowApplication.Completed%2A>chiamato una volta completato il processo di annullamento.</xref:System.Activities.WorkflowApplication.Completed%2A>"
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is cancelled.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#8](~/add/codesnippet/csharp/t-system.activities.unha_3.cs)]"
  syntax:
    content: Cancel
    return:
      type: System.Activities.UnhandledExceptionAction
      description: "Da aggiungere."
  exceptions: []
  platform:
  - net462
- uid: System.Activities.UnhandledExceptionAction.Terminate
  id: Terminate
  parent: System.Activities.UnhandledExceptionAction
  langs:
  - csharp
  name: Terminate
  nameWithType: UnhandledExceptionAction.Terminate
  fullName: System.Activities.UnhandledExceptionAction.Terminate
  type: Field
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Specifica che il <xref href=&quot;System.Activities.WorkflowApplication&quot;> </xref> deve pianificare la chiusura dell&quot;attività radice e riprendere l&quot;esecuzione."
  remarks: "Di conseguenza, <xref:System.Activities.WorkflowApplication.Completed%2A>chiamato una volta completato il processo di terminazione.</xref:System.Activities.WorkflowApplication.Completed%2A> L&quot;eccezione non gestita viene utilizzata come motivo di interruzione. Terminare è l&quot;azione predefinita se non <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>gestore viene specificato.</xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>"
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is terminated.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#6](~/add/codesnippet/csharp/t-system.activities.unha_1.cs)]"
  syntax:
    content: Terminate
    return:
      type: System.Activities.UnhandledExceptionAction
      description: "Da aggiungere."
  exceptions: []
  platform:
  - net462
references:
- uid: System.Enum
  isExternal: false
  name: System.Enum
- uid: System.Activities.UnhandledExceptionAction.Abort
  parent: System.Activities.UnhandledExceptionAction
  isExternal: false
  name: Abort
  nameWithType: UnhandledExceptionAction.Abort
  fullName: System.Activities.UnhandledExceptionAction.Abort
- uid: System.Activities.UnhandledExceptionAction
  parent: System.Activities
  isExternal: false
  name: UnhandledExceptionAction
  nameWithType: UnhandledExceptionAction
  fullName: System.Activities.UnhandledExceptionAction
- uid: System.Activities.UnhandledExceptionAction.Cancel
  parent: System.Activities.UnhandledExceptionAction
  isExternal: false
  name: Cancel
  nameWithType: UnhandledExceptionAction.Cancel
  fullName: System.Activities.UnhandledExceptionAction.Cancel
- uid: System.Activities.UnhandledExceptionAction.Terminate
  parent: System.Activities.UnhandledExceptionAction
  isExternal: false
  name: Terminate
  nameWithType: UnhandledExceptionAction.Terminate
  fullName: System.Activities.UnhandledExceptionAction.Terminate
